crates/openapi/src/routes/customers.rs

```diff
@@ -0,0 +1,101 @@
+/// Customers - Create
+///
+/// Creates a customer object and stores the customer details to be reused for future payments. Incase the customer already exists in the system, this API will respond with the customer details.
+#[utoipa::path(
+    post,
+    path = "/customers",
+    request_body  (
+        content = CustomerRequest,
+      examples  (( "Update name and email of a customer" =(
+        value =json!( {
+            "email": "guest@example.com",
+            "name": "John Doe"
+        })
+      )))
+    ),
+    responses(
+        (status = 200, description = "Customer Created", body = CustomerResponse),
+        (status = 400, description = "Invalid data")
+
+    ),
+    tag = "Customers",
+    operation_id = "Create a Customer",
+    security(("api_key" = []))
+)]
```

Comment:
Keep the syntax consistent. Ensure this for all `utoipa::path` invocations.
```suggestion
#[utoipa::path(
    post,
    path = "/customers",
    request_body  (
      content = CustomerRequest,
      examples(("Update name and email of a customer" = (
        value = json!({
            "email": "guest@example.com",
            "name": "John Doe"
        })
      )))
    ),
    responses(
        (status = 200, description = "Customer Created", body = CustomerResponse),
        (status = 400, description = "Invalid data")
    ),
    tag = "Customers",
    operation_id = "Create a Customer",
    security(("api_key" = []))
)]
```